dbt/README.md

```diff
@@ -571,7 +624,7 @@ the tag `test_qc_iasworld`
     * `category` (optional): A workbook category for the test, required if
       a category is not defined for the test's generic in the `TEST_CATEGORIES`
       constant in the [`transform_dbt_test_results`
-      script](https://github.com/ccao-data/data-architecture/blob/master/.github/scripts/transform_dbt_test_results.py)
+      script](https://github.com/ccao-data/data-architecture/blob/master/dbt/scripts/transform_dbt_test_results.py)
```

Comment:
**nitpick (non-blocking):** A super minor thing, but IMO relative URLs are better for things inside the repo, just in case the path/owner ever changes. It also lets you navigate the web UI correctly while in a branch other than master.

Reply:
Great call, I switched all of the links to code in the README to use relative paths in 7d20c44b3e20d39367c3f026fd1d350c645ecda5.